src/components/binding/binding-panel.js

```diff
@@ -213,13 +187,44 @@ genotet.BindingPanel.prototype.updateTracks = function() {
       this.container.find('#genes #track-' + index).remove();
     }
   }
+
+  this.signal('loadBindingList');
+
+  this.container.find('#genes .glyphicon-remove')
+    .css('display', this.data.tracks.length == 1 ? 'none' : '');
+};
+
+/**
+ * Updates the track list for panel.
+ */
+genotet.BindingPanel.prototype.updateTrackList = function() {
```

Comment:
No, they cannot be combined. Because we should load binding list before updateTrackList and the signal 'addPanelTrack' cannot call loadBindingList directly (you pointed out before, because it is not illogical). 'addPanelTrack' can only call a function like updateTrack.

By the way, why do you always point out some illogical designs and point out that they are illogical again after I revised them to the way you think is logical? I feel really tired to change something reduplicatively.